react/src/components/Boxes.module.css

```diff
@@ -0,0 +1,356 @@
+    text-align: left;
+    letter-spacing: 0;
+}
+
+#centerBoxHeader {
+    /* Self Layout Properties */
+    width: 14.3125rem;
+    height: 1.8125rem;
+
+    /* UI Properties */
+    font-size: 1.5rem;
+    font-family: 'Lato', sans-serif;
+    color: #428E9E;
+}
+
+#centerBoxText {
+    /* Self Layout Properties */
+    width: 17.25rem;
+    height: 1.1875rem;
+
+    /* UI Properties */
+    color: #333;
+}
+
+@media only screen and (max-width: 62rem) {
+    .pDesktopSubBox {
+        display: none;
+    }
+}
+
+@media only screen and (max-width: 37.5rem) {
+    #mainBox {
+        /* Self Layout Properties */
+        padding: 40px 0;
+
+        /* Content Layout Properties */
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+
+    #container {
+        /* Self Layout Properties */
+        width: 93.75%;
+        height: 75%;
+
+        /* Content Layout Properties */
+        flex-wrap: wrap;
+        justify-content: space-between;
+    }
+
+    .subBox {
+        /* UI Properties */
+        font-family: 'Lato', sans-serif;
+    }
+
+    #centerBoxContainer {
+        /* Self Layout Properties */
+        width: 100%;
+        padding: 10rem 0;
+        order: 0;
+    }
+
+    #centerBoxOutline {
+        /* Self Layout Properties */
+        position: absolute;
+        padding: 10rem 0;
+    }
+
+    #centerBox {
+        /* Self Layout Properties */    
+        width: calc(100% - 0.125rem);
+        height: calc(100% - 0.125rem);
+        z-index: 1;
+    }
+
+    #centerBoxInner {
+        /* Self Layout Properties */
+        height: 100%;
+        width: 100%;
+
+        /* UI Properties */
+        background: transparent url('../icons/smartphone/Pattern.png') bottom no-repeat padding-box;
+    }
+
+    #centerBoxInner p {
+        /* Self Layout Properties */
+        padding: 0 0.547rem;
+
+        /* UI Properties */
+        font-size: 0.75rem;
+    }
+
+    #centerBoxHeader {
+        /* UI Properties */
+        line-height: 1.5;
+    }
+
+    #centerBoxText {
+        line-height: 3;
+        font-family: 'Lato Light', sans-serif;
+    }
+
+    #topLeftBox, #topRightBox, #bottomLeftBox, #bottomRightBox {
+        /* Content Layout Properties */
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+
+    #topLeftBox p, #topRightBox p, #bottomLeftBox p, #bottomRightBox p {
+        /* UI Properties */
+        font-size: 1.875rem;
+    }
+
+    #topRightBox, #bottomLeftBox, #bottomRightBox {
+        /* Self Layout Properties */
+        padding: 0.75rem 1.331rem;
+    }
+
+    #topLeftBoxContainer {
+        /* Self Layout Properties */
+        order: 1;
+        position: relative;
+        padding: 1.875rem 1.875rem;
+    }
+
+    #topLeftBox {
+        /* Self Layout Properties */
+        margin-top: -0.125rem;      /* Change to 0 when inactive */
+        padding: 0.75rem 1.2688rem; /* Change to 0.6875rem 1.2688rem when inactive */
```

Comment:
Zbyt dokładnie są te remy dodane, trzymajmy się prostych wartości, np. `1.2rem`, `1.3rem`. `0.75rem` to max + przypominam o nie ustawianiu szerokości i wysokości na sztywno. (nie musisz tego już poprawiać)

Reply:
Ah ten mój obsesyjny perfekcjonizm...

Wszystkie prócz jednej sztywnej szerkości/wysokości wyrzuciłem, ale chyba nie zapisałem zmian. 😅

Pozostała mi tylko szerokość samego komponentu. Generalnie komponent wygląda tak: (border dodałem dla wizualizacji).
![obraz](https://user-images.githubusercontent.com/108541084/179232834-47a20aa1-7d6b-41a0-b510-e0c45375252c.png)
W teorii mógłbym go rozciągnąć na szerokość ekranu (oczywiście z uwzględnieniem paddingu), ale tak chyba też jest w porządku?